### PR TITLE
Make scraper write directly to DB

### PR DESCRIPTION
**Pull request by tomquisel**
- Enable mode that uses `sqlalchemy` to write directly to DB
- Make the scraper more configurable via environment variables 
- Some annoying autoformatting

Note: GraphQL writing may be broken now, I haven't tested it thoroughly.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,15 +1,11 @@
-FROM python:3.5-alpine
+# Using debian buster because this: https://pythonspeed.com/articles/base-image-python-docker-images/
```
Comment (dstuck) [inline Dockerfile:1]: Thanks for the update and reference here!

**Review by dstuck — APPROVED** [review]
Nice! Happy to see a pretty straightforward implementation of the direct sql store.

**Comment by tomquisel** [comment]
Thanks! Feel free to merge if you're happy with it, I don't have permissions to do it.

**Comment by dstuck** [comment]
@tomquisel Oh, I've added you as a collaborator, but it says you haven't accepted. This should link there: https://github.com/dstuck/clinical-trials-api-scraper/invitations

**Comment by tomquisel** [comment]
Oops! Thanks :)

**Pull request merged**